Scripts/Runtime/Entities/PersistentData/Data/EntityPersistentData.cs

```diff
@@ -97,18 +90,19 @@ public JobHandle MigrateTo(JobHandle dependsOn, IMigratablePersistentData destin
             EntityPersistentData<T> destinationEntityPersistentData = (EntityPersistentData<T>)destinationPersistentData;
 
             //Launch the migration job to get that burst speed
-            dependsOn = JobHandle.CombineDependencies(dependsOn,
-                AcquireWriterAsync(out EntityPersistentDataWriter<T> currentData),
-                destinationEntityPersistentData.AcquireWriterAsync(out EntityPersistentDataWriter<T> destinationData));
+            dependsOn = JobHandle.CombineDependencies(
+                dependsOn,
+                AcquireSharedWriteAsync(out EntityPersistentDataWriter<T> currentData),
+                destinationEntityPersistentData.AcquireSharedWriteAsync(out EntityPersistentDataWriter<T> destinationData));
```

Comment:
I didn't notice this before because of the API, but these should really be Exclusive. Is it a huge pain to add the Exclusive support now?

Reply:
Fixed